hypha/apply/funds/tables.py

```diff
@@ -443,3 +443,18 @@ class Meta:
         order_by = ('-created_at',)
         attrs = {'class': 'all-reviews-table'}
         empty_text = _('No reviews available')
+
+
+class StaffAssignmentsTable(tables.Table):
+    full_name = tables.LinkColumn('funds:submissions:list', orderable=True, verbose_name="Staff", attrs={'td': {'class': 'title'}})
+    reviewer_roles = ReviewerRole.objects.all().order_by('order')
+    role1 = tables.Column(verbose_name=reviewer_roles.first())
+    role2 = tables.Column(verbose_name=reviewer_roles.last())
```

Comment:
This code will run only when the code loads but will not update when data changes in db. We may not need this by using `extra_columns`. More details in another comment.